src/test/java/com/epam/tc/hw5/ex1/steps/GivenStep.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.tc.hw5.ex1.steps;
+
+import com.epam.tc.hw5.ConfProperties;
+import com.epam.tc.hw5.ex1.cucumber.context.TestContext;
+import io.cucumber.java.en.Given;
+
+public class GivenStep extends AbstractStep {
```

Comment:
you don't have to create separated GivenStepS classes for each exercise. fix it.